nft_ingester/src/bin/ingester/main.rs

```diff
@@ -800,6 +800,31 @@ pub async fn main() -> Result<(), IngesterError> {
         batch_mint_persister.persist_batch_mints(rx).await
     });
 
+    // clean indexes
+    for asset_type in ASSET_TYPES {
+        let primary_rocks_storage = primary_rocks_storage.clone();
+        let mut rx = shutdown_rx.resubscribe();
+        mutexed_tasks.lock().await.spawn(async move {
+            loop {
+                if rx.try_recv().is_ok() {
+                    break;
+                }
+
+                match clean_syncronized_idxs(primary_rocks_storage.clone(), asset_type) {
+                    Ok(_) => {
+                        info!("Cleaned synchronized indexes for {:?}", asset_type);
+                    }
+                    Err(e) => {
+                        error!("Failed to clean synchronized indexes for {:?} with error {}", asset_type, e);
+                    }
+                }
+                tokio::time::sleep(Duration::from_secs(SECONDS_TO_RETRY_IDXS_CLEANUP)).await;
```

Comment:
how does it work with termination (ctrl+c) of the app? we usually switch on it and the shutdown_rx